### ISSUE TITLE
Incoorporate AllowMultipleGestureRecognizer with GestureDetector

### ISSUE THREAD

**Issue by TomOerlemans** [flutter/flutter#48715]
## Use case

Apps run into situations where overlapping widgets need to respond to gestures, e.g.: a calendar app, which is able to both drag calendar items, as well as scroll through a list of those calendar items.

## Proposal

Currently the only way I am able to get this functionality is through "AllowMultipleGestureRecognizer" (see: https://www.davidanaya.io/flutter/combine-multiple-gestures.html). I find this method rather complex, and requiring a lot of boilerplate code. If it is possible to incorporate this into the basic functionality of a GestureDetector, it would make things a lot easier in my opinion.

**Comment by renenucci** [comment]
I Agree, I'm trying to implement "two fingers" bottom to up event with multidrag and RawGestureDetector but seems too complex to "simple" UI interaction, would be nice if GestureDetector encapsulates new multiple gestures.
